### PR TITLE
chore: always use prefix in dev storybook

### PR DESCRIPTION
**Pull request by lee-chase**
Contributes to #1154 

Propose changing all development storybook stories to use a custom prefix.

#### What did you change?

Core storybook preview and styling to set a prefix when in development mode.

#### How did you test and verify your work?

- yarn storybook
- Viewed in storybook and inspected the DOM and noted prefix
- yarn build
- npx serve packages/core/storybook-static and checked the prefix in the DOM

### CODE REVIEW

File: packages/core/.storybook/preview.js [hunk 1]
```diff
@@ -38,6 +39,12 @@ const Style = ({ children, styles }) => {
   return children;
 };
 
+let isDev = CONFIG_TYPE === 'DEVELOPMENT'; //  process.env?.NODE_ENV === 'development';
```
Comment (matthewgallo) [inline packages/core/.storybook/preview.js:42]: Don't think you need to use `let` here. Also, where is `CONFIG_TYPE` coming from? 🤔

Comment (matthewgallo) [reply packages/core/.storybook/preview.js:42]: We have a `isProduction` variable that could possibly be used from `packages/ibm-products/src/global/js/utils/pconsole.js`.

Comment (lee-chase) [reply packages/core/.storybook/preview.js:42]: As far as I can tell Storybook does not pass the NODE_ENV parameter

Comment (lee-chase) [reply packages/core/.storybook/preview.js:42]: CONFIG_TYPE is added by storybook I believe

https://github.com/search?q=repo%3Astorybookjs%2Fstorybook%20config_type&type=code

**Review by matthewgallo — APPROVED** [review]

**Pull request merged**